tests/testthat/test-as_html.R

```diff
@@ -1,6 +1,9 @@
 context("Exporting to HTML")
 
 test_that("as_html smoke test", {
+  require(xml2, quietly = TRUE)
+  skip_if_not_installed("xml2")
+
```

Comment:
In general it's better to return early before executing (independent) commands

```suggestion
  skip_if_not_installed("xml2")
  require(xml2, quietly = TRUE)

```

Reply:
@shajoezhu this is what I mentioned in the last PRs ;)